### DISCUSSION TITLE
Community Discord Server

### DISCUSSION THREAD

**Discussion by Shinigami92** [lodash/lodash#5722] [Ideas]
Build up a Community Discord Server

Big Open Source Projects commonly uses e.g. https://chat.lodash.com/ to link to their Discord Server

Example: https://chat.vitejs.dev/, https://chat.fakerjs.dev/

- [x] I'm willing to help to initially help moderating and organizing the server a bit

**Comment by jdalton** [comment]
I was just looking for something like this. Discord does seem to be the way nowadays.

**Reply by Shinigami92** [reply]
Oh, :laughing: I just recognized that I accidentally didn't check the checkbox for the "willing to help" lol
That checkbox was meant to indicate that I WANT TO HELP with setting up the Discord server :rofl: 

I can't send you my account name here in public, I think I will send you somehow a DM so you can invite me on Discord and then I can start (tomorrow) to help setup the server

I do not have twitter verification, so I cant send you a message there :(
Could you e.g. join the Faker Discord server and there ping me? ^^'

**Reply by Shinigami92** [reply]
@jdalton I added you on Discord (found you via the Bun Discord server)

I would be happy to start kick-off a lodash community server
I just want that you are the owner but I can be a moderator/community manager (in my free time)
